provisioners.proto

```diff
@@ -132,8 +132,16 @@ message AzureProvisioner {
 }
 
 message ACMEProvisioner {
+	enum Challenge {
```

Comment:
Would go for `ChallengeType`. Not putting it as a suggestion, as the `pb` has to be regenerated.

Reply:
Fixed with 6d52d6e